Content.Shared/Cloning/CloningPodComponent.cs

```diff
@@ -0,0 +1,137 @@
+using Content.Shared.Construction.Prototypes;
+using Content.Shared.Materials;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Cloning;
+[RegisterComponent]
+public sealed partial class CloningPodComponent : Component
+{
+    public const string PodPort = "CloningPodReceiver";
+
+    [ViewVariables]
+    public ContainerSlot BodyContainer = default!;
+
+    /// <summary>
+    /// How long the cloning has been going on for.
+    /// </summary>
+    [ViewVariables]
+    public float CloningProgress = 0;
```

Comment:
Not sure if it's in scope of this PR, but this should probably be changed to `TimeSpan` datafield for when the cloning is done, with `TimeOffsetSerializer`. `CloningTime` and `BaseCloningTime` should be `TimeSpan` also.

Reply:
Not in the scope of this PR, it's only a conversion and not a refactoring.